src/components/layout.scss

```
@@ -20,6 +20,31 @@ h1, h2, h3, h4, h5, h6, li, p, span, a {
   transform: rotate(0.03deg);
 }
 
+//もしページの内容が画面高さより小さくても、body要素は最低でも画面高さを持つようにします。
+html {
+  height: -webkit-fill-available;
+}
+body {
+  min-height: 100vh;
+  min-height: -webkit-fill-available;
+}
+
+//footerを最下部に固定します。
+body {
+  position: relative;
+  .footer {
+    &.appearance {
+      position: absolute;
+      bottom: 0;
+      width: 100%;
+    }
+    &.height-holder {
+      opacity: 0;
+      pointer-events: none;
+    }
```

`opacity: 0;` は要素がアクセシビリティツリーから削除されないので`visibility: hidden;`のほうが適切だと感じたのですが、いかがでしょうか？
```suggestion
    &.height-holder {
      visibility: hidden;
    }
```

---

ご指摘のとおりです！ありがとうございます！